helloWorld/.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```

Comment:
You should not create .idea folder. This kind of folder is being automatically generated by IntellJ Idea when you create a project. So, it should be rather ignored. Please move your gitignore to root